tests/native_api_tests/test_api_model.cpp

```diff
@@ -337,17 +350,22 @@ TEST_CASE("API Model") {
             options.set_calculation_method(PGM_iterative_current);
             try {
                 model.calculate(options, single_output_dataset);
+                throw std::exception{};
             } catch (PowerGridRegularError const& e) {
                 check_exception(e, PGM_regular_error, "The calculation method is invalid for this calculation!"s);
             }
         }
 
         SUBCASE("Batch calculation error") {
             // wrong id
-            load_updates_id[1] = 5;
+            load_updates_id[1] = 98;
```

Comment:
You don't like `99`?

Reply:
`99` is still used in the "update error" subcase. I just wanted to use a different number for the other test case :)